app/javascript/components/toast-list/toast-list.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  ToastNotificationList, TimedToastNotification,
+} from 'patternfly-react';
+import { markNotificationRead, removeToastNotification } from '../../miq-redux/actions/notifications-actions';
+
+const ToastList = () => {
+  const dispatch = useDispatch();
+  const toastNotifications = useSelector(({ notificationReducer: { toastNotifications } }) => toastNotifications);
+  const notificationTimerDelay = 8000;
```

Comment:
I would define this constant outside of the component. If you leave it is at is, it will create a new variable for each notification which uses slightly more memory. It's not a big deal but its a good practice.